### ISSUE TITLE
How to fine tuning model on multi-tasks?

### ISSUE THREAD

**Issue by swufelyc** [xuyige/BERT4doc-Classification#3]
It seems to me, the run_classifier_single_layer.py does not save the model, and what should I do to fine tuning the fine tuned model?

**Comment by xuyige** [comment]
Thank you for your issue!
I have just uploaded our codes about the fine-tuning model on multi-tasks.
The multi-task fine-tuning is just adding other softmax layers for other tasks.
In our paper, only four datasets are used.

**Comment by xuyige** [comment]
For saving models, we did not save checkpoints during fine-tuning. If you need to save your models, we suggest using torch.save

**Comment by swufelyc** [comment]
Ok, thanks for your reply！
